src/databricks/labs/ucx/workspace_access/groups.py

```diff
@@ -38,6 +38,7 @@ class MigratedGroup:
     entitlements: str | None = None
     external_id: str | None = None
     roles: str | None = None
+    is_same_membership: bool | None = None
```

Comment:
no, you're not allowed to add this column, as it'll be a breaking change to database, which this small feature is not justifying

Reply:
Changed the logic to incorporate it with CLI.